src/main/java/Exceptions.java

```diff
@@ -20,6 +20,10 @@
 
     public final static String INVALID_AMOUNT = "Please enter a valid integer between 1 and 1000000.";
     public final static String INSUFFICIENT_BALANCE = "Insufficient balance. Please load first.";
+    
+    // error messages associated with point redemption.
+    
+    public final static String INSUFFICIENT_POINTS = "Insufficient points to redeem."
```

Comment:
Yes, with this, we can display this message in upper level class if customer want to redeem a rewards item but has insufficient points. Nice to add it.

Reply:
BTW at your line 26, you forget to have ;

Reply:
Nice catch! Got it!